README.md

```diff
@@ -1,3 +1,7 @@
+> NOTE: These sample applications use the Programmable Voice Android SDK 2.x APIs. For examples
+using our 3.x APIs, please see the [master](https://github.com/twilio/voice-quickstart-android)
+branch.
```

Comment:
@aaalaniz Im planning to add the following text to the change logs and over view
> If you are already using 2.x, we highly recommend planning your migration to 3.0 as soon as possible. Version 2.x is now deprecated and will only receive fixes for critical and security related issues. Support for 2.x will cease 1/1/2020.

Reply:
I'll update the note with a recommendation to upgrade, a link to our migration guide and notice about deprecation